staging/src/k8s.io/client-go/tools/clientcmd/api/v1/conversion.go

```diff
@@ -167,6 +168,9 @@ func addConversionFuncs(scheme *runtime.Scheme) error {
 				if err := s.Convert(&curr.Context, newContext, 0); err != nil {
 					return err
 				}
+				if len(curr.Name) == 0 {
+					return fmt.Errorf("you must specify a non-empty context name for %#v", curr)
```

Comment:
non-empty context names are not allowed. "you" sounds strange here.

Reply:
Make sense. Updated. PTAL.